test/config.test.ts

```diff
@@ -51,6 +51,7 @@ function assertSchema(
         const validator = new Ajv({
             // skip full check if we are not encoding refs
             validateFormats: config.encodeRefs === false ? undefined : true,
+            allowUnionTypes: true,
```

Comment:
Why is this change necessary for windows support?

Reply:
Sorry - I've been flying fast and loose. Will keep it clean going forward